js/scripts.js

```diff
@@ -0,0 +1,46 @@
+var a, b, value, switcher;
+
+a=prompt ("Podaj a:");
+b=prompt ("Podaj b:");
+			
+value = (a*a) + (2*a*b) - (b*b);
+		
+console.log ('Wynik operacji wynosi: ' + value);
+/*
+if (value>0){ 
+	console.log ('Wynik dodatni');
+}
+		
+if (value<0){
+	console.log ('Wynik ujemny');
+}
+
+if (value==0){
+	console.log ('Wynik równy zero');
+}
+*/
+/*
+if (value>0){ 
+	console.log ('Wynik dodatni');
+} else if (value<0){
+	console.log ('Wynik ujemny');
+} else if (value==0){
+	console.log ('Wynik równy zero');
+}
+
+*/
+
+switcher = (value<0)*(-1) + (value==0)*(0) + (value>0)*(1);
```

Comment:
switcher out - to jest jakaś rzeźnia - operowanie true / false + operacje arytmetyczne na nich - to prosi się o błędy w logice - nie nie nie :D